Google.PowerShell/Sql/GcSqlSslCmdlets.cs

```diff
@@ -479,6 +480,10 @@ protected override void ProcessRecord()
                 default:
                     throw UnknownParameterSetException;
             }
+            if (!ShouldProcess($"{project}/{instance}", "Reset SSL Certificate Configuration"))
+            {
+                return;
```

Comment:
Using  `if (!X) { return; }` should only be reserved if the following code block is very long. Since it is only 4-lines, I'd just put it in an `if (ShouldProcess(...)) {` block.